packages/valory/skills/abstract_round_abci/test_tools/rounds.py

```diff
@@ -576,7 +576,7 @@ def _test_round(
         for behaviour_attr_getter in synchronized_data_attr_checks:
             assert behaviour_attr_getter(synchronized_data) == behaviour_attr_getter(
                 actual_next_synchronized_data
-            )
+            ), f"Mismatch in synchronized_data. Actual:\n{behaviour_attr_getter(synchronized_data)}\nExpected:\n{behaviour_attr_getter(actual_next_synchronized_data)}"
```

Comment:
Words and variable names don't match..

Reply:
I think there's a naming problem here: `synchronized_data` is the result of `end_round()` and `actual_next_synchronized_data` the result of calling `synchronized_data_update_fn()` with `final_data`, so if I understand it correctly here actual = expected.